demo/main.py

```diff
@@ -95,31 +87,15 @@ def main(
 # === ASYNC EXAMPLE ===
 
 
-async def async_play_audio(data: AsyncGenerator[bytes, None] | AsyncIterable[bytes]):
-    buff_size = 10485760
-    ptr = 0
-    start_time = time.time()
-    buffer = np.empty(buff_size, np.float16)
-    audio = None
+async def async_save_audio(data: AsyncGenerator[bytes, None] | AsyncIterable[bytes]):
```

Comment:
I see you simplified the demo by just saving the audio and not dealing with the more complex -> playing of it, right ?

Reply:
Yes, I was having a lot of dependency version issues with simpleaudio. Not the first time I have seen this. Think it is easier to just save it.

Reply:
Agreed